docs/configuration.md

```diff
@@ -26,13 +26,69 @@ variables.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_REDIS_USERNAME
+
+- Redis ACL user name
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_PASSWORD
+
+- Redis ACL password
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_REDIS_SSL
+
+- Whether use SSL to connect to Redis
+- Optional. Defaults to empty.
```

Comment:
boolean defaults to `empty`?

Reply:
Yes, it defaults to not be set.

Reply:
maybe 
```suggestion
- Optional. Defaults to not set, in which case URI schema will be used to decide.
```

Reply:
That's fine, but it's currently not true for apisonator, I can fix it, it shouldn't be too hard.

Reply:
If possible, I'd suggest remove this setting and use schema. The less configuration, the better.

Reply:
Ok, let's discuss there.